src/pages/namespace/Explorer/Workflow/Active/WorkflowEditor.tsx

```diff
@@ -7,34 +6,48 @@ import {
   DropdownMenuTrigger,
 } from "~/design/Dropdown";
 import { FC, useEffect, useState } from "react";
-import { Popover, PopoverContent, PopoverTrigger } from "~/design/Popover";
+import { GitBranchPlus, Play, Save, Tag, Undo } from "lucide-react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "~/design/Tooltip";
+import {
+  availableLayouts,
+  layoutIcons,
+  useEditorActions,
+  useEditorLayout,
+} from "~/util/store/editor";
 
 import Button from "~/design/Button";
 import { ButtonBar } from "~/design/ButtonBar";
-import { Card } from "~/design/Card";
-import Editor from "~/design/Editor";
+import { CodeEditor } from "./CodeEditor";
+import { Diagram } from "./Diagram";
 import RunWorkflow from "../components/RunWorkflow";
 import { RxChevronDown } from "react-icons/rx";
+import { Toggle } from "~/design/Toggle";
+import { WorkspaceLayout } from "../../../../../componentsNext/WorkspaceLayout";
 import { useCreateRevision } from "~/api/tree/mutate/createRevision";
 import { useNodeContent } from "~/api/tree/query/node";
 import { useRevertRevision } from "~/api/tree/mutate/revertRevision";
-import { useTheme } from "~/util/store/theme";
 import { useTranslation } from "react-i18next";
 import { useUpdateWorkflow } from "~/api/tree/mutate/updateWorkflow";
-import useUpdatedAt from "~/hooksNext/useUpdatedAt";
 
-// get type of useNodeContent return value
-type NodeContentType = ReturnType<typeof useNodeContent>["data"];
+export type NodeContentType = ReturnType<typeof useNodeContent>["data"];
 
 const WorkflowEditor: FC<{
   data: NonNullable<NodeContentType>;
   path: string;
 }> = ({ data, path }) => {
+  const layout = useEditorLayout();
+  const { setLayout } = useEditorActions();
```

Comment:
I had first commented in line 103. It is probably best to read that comment first to understand this suggestion.

After thinking about this for a bit, I think it is often better to have slightly more elaborate names for variables. Especially when they are in the broader scope if an entire component. A single word variable name is often not very obvious to understand when read out of context, and hard to differentiate from other variables that refer to the same type of entity.

2 word variable names, combining an adjective+noun, are often much easier to understand.

```
  const currentLayout = useEditorLayout();
  const { setLayout: setCurrentLayout } = useEditorActions();
```

Reply:
I totally agree with that. Simple and short variables are nice to ready, but they should not be too generic so that they are not understandable. (fixed in 355912463ccaaafda277d62a0c65a2bfab2387f4)